ArcSlider/ArcSlider.js

```diff
@@ -240,8 +261,8 @@ const ArcSliderBase = kind({
 					<circle
 						cx={knobPosition.x}
 						cy={knobPosition.y}
-						fill={isFocused ? accentColor : foregroundColor}
-						r={ri.scaleToRem(15)}
+						fill={!noFocusColor && isFocused ? accentColor : foregroundColor}
```

Comment:
Could you explain why we need noFocusColor prop? it looks always true.

Reply:
@jeonghee27 
With the previous implementation, the knob color would change to the accent color of the skin on hover. To keep that functionality I added the 'noFocusColor' prop to be able to use that previous feature, but made the default to not use it. If you think it's not required, I will remove it. What do you think?

Reply:
Ok.
Since there is no GUI about the knob color, I think it is not necessary to support it. If there is a need later,  even if it is implemeted againt at that time.